lib/prelude.dx

```diff
@@ -249,6 +249,9 @@ def withState
     def explicitAction (h':Type) (ref:Ref h' s) : {State h'|eff} a = action ref
     %runState init explicitAction
 
+def unsafeIO (f: Unit -> {State World|eff} a) : {|eff} a =
```

Comment:
While `State World` is a cute way to represent IO, I think it would be much easier to understand for people if we could just alias that to the name `IO`

Reply:
This still has to be updated 😕